frontend/www/js/omegaup/components/course/CardsList.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <div>
+    <h1 class="card-title">{{ T.navAllCourses }}</h1>
+
+    <div class="m-3" v-if="false">
+      <div class="float-right">
+        <a class="btn btn-primary" href="/course/new/">{{ T.courseNew }}</a>
+      </div>
+      <h1>&nbsp;</h1>
+    </div>
+    <div class="container">
+      <div
+        class="row"
+        v-for="(typeCourses, accessMode) in courses"
+        v-if="typeCourses.activeTab !== ''"
+      >
+        <div class="col-lg-5 p-3" v-bind:class="accessMode">
+          <div class="float-left">
+            <h3>{{ getDescription(accessMode) }}</h3>
+          </div>
+          <div class="float-right">
+            <font-awesome-icon icon="info-circle" />
+          </div>
+        </div>
```

Comment:
esto funciona?

```suggestion
        <div class="col-lg-5 p-3 d-flex" v-bind:class="accessMode">
          <h3 class="flex-grow-1">{{ getDescription(accessMode) }}</h3>
          <div>
            <font-awesome-icon icon="info-circle" />
          </div>
        </div>
```

preferiría evitar el uso de `float` de ser posible.

además, el `info-circle` parece que no está desplegando nada. se le puede tener un texto explicatorio  o eliminarlo?

Reply:
@tvanessa sabes si ya se tiene la información que desplegará el icono? de otra forma, habrá que quitarlo hasta que ya se tenga listo

Reply:
Por lo pronto quitaré el icono